app/helpers/flood_risk_engine/email_helper.rb

```diff
@@ -0,0 +1,20 @@
+module FloodRiskEngine
+  module EmailHelper
+
+    # Embed an image inline into html email
+    def email_image_tag(image, **options)
+      path = "/app/assets/images/#{image}"
+
+      full_path = Rails.root.join path
```

Comment:
As you are effectively using a File join on line 8, I'd suggest letting that do the all the concatenation work, and slash insertion. Something like:

```
image_folder = "app/assets/images"
path = Rails.root.join image_folder, image
```